gulpfile.mjs

```diff
@@ -157,7 +157,7 @@ gulp.task(
       bootstrapFile: 'src/sidebar/test/bootstrap.js',
       karmaConfig: 'src/karma.config.js',
       rollupConfig: 'rollup-tests.config.mjs',
-      testsPattern: 'src/**/*-test.js',
+      testsPattern: 'src/**/*-test.{js,ts,tsx}',
```

Comment:
I added tsx extension, as we'll need it as soon as a component's test is migrated to TS.